app/jobs/back_in_stock_notification_mailer_job.rb

```diff
@@ -0,0 +1,21 @@
+class BackInStockNotificationMailerJob < ApplicationJob
+
+  def perform(email:, stock_location:)
+    send_email(email, stock_location)
```

Comment:
Totally minor and definitely a matter of preference, but if a method does nothing but call a different private method, I tend to think it isn't adding much and would prefer the code all just be in the public method.

Reply:
Ok, I'm happy to move it back. The only reason I moved it into a separate method was because I wasn't sure if the job might retry if I run `return unless user_bisns.present?` within the perform method. I guess not!

Reply:
Ah yeah, they only retry if there's an uncaught exception.